golang/config/environ.sls

```diff
@@ -27,3 +27,16 @@ golang-config-file-managed-environ_file:
         golang: {{ golang|json }}
     - require:
       - sls: {{ sls_archive_install if golang.pkg.use_upstream_archive else sls_package_install }}
+
+
+    {% for i in ('go', 'godoc', 'gofmt') %}
+
+golang-config-file-symlink-golang-binaries:
```

Comment:
You're going to need to reference `i` here otherwise you'll end up with duplicate state IDs.  So something like:

```suggestion
golang-config-file-symlink-golang-binaries-{{ i }}:
```

Reply:
Thank you @myii